.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,21 @@
-## What is the purpose of this change? What does it change?
-<!-- Describe the changes here, as detailed as needed. -->
+**What kind of PR is this?**
+<!--
+DELETE the kind(s) which are not applicable before opening the PR.
+-->
 
-## Was the change discussed in an issue?
-fixes #???
-<!-- Please do Link issues here. -->
+/kind api-change
+/kind bug
+/kind cleanup
+/kind code refactorting
+/kind docs
+/kind enhancement
+/kind feature
+/kind flake
```

Comment:
Note: The reason why we do NOT put these in the commented portion, like so:

```sh
<!--
/kind cleanup
/kind code refactorting
/kind docs
/kind enhancement
/kind feature
/kind flake
-->
```

Is because @openshift-ci-robot will *still* apply the label despite it being inside of a comment..

Hence why there is a comment that says to delete what you dont need :)